estate/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
We try to include the return character on the last line in every file.
This way when adding more lines later, the previously last line is not impacted in the diff.